apps/condo/domains/contact/components/ContactsEditor.tsx

```diff
@@ -0,0 +1,556 @@
+import React, { useCallback, useState, useMemo, useRef, useEffect } from 'react'
```

Comment:
Spread components across files, impossible to read

Reply:
Splitted in `2ccbee7e`